lib/modules/versioning/ruby/index.ts

```diff
@@ -19,7 +19,6 @@ export const id = 'ruby';
 export const displayName = 'Ruby';
 export const urls = [
   'https://guides.rubygems.org/patterns/',
-  'https://bundler.io/v1.5/gemfile.html',
```

Comment:
why removed?

Reply:
This link keeps changing, let's leave it off